app/client/templates/components/app-item.scss

```diff
@@ -136,8 +136,8 @@
         display: block;
         font-size: $app-install-font-size;
         font-weight: normal;
-        padding-top: $app-install-padding-tb;
-        padding-bottom: $app-install-padding-tb;
+        padding-top: 0.6rem;
```

Comment:
General question: is `$app-install-padding-tb` used anywhere now? if not, can we delete the constant?

When I went to look into that, I noticed that we seem to have both `app-item.scss` and `app-item.scss.flex`. Is one of these unused? Do you happen to know? :) I can try to find out, but if you can outright remove one of these files, then that'd be great by me!

Another approach would be to change the constant to `0.6rem` and rename it to `$app-install-padding-top`.

Curious if you have an opinion about the above thoughts.